.github/workflows/android-test.yml

```diff
@@ -0,0 +1,44 @@
+name: android-test
+on:
+  push: # Since it takes time and is unstable, execute only when merging
+    branches:
+      - main
+
+jobs:
+  test:
+    name: "android test on main branch"
+    runs-on: macos-latest # To use emulator
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        # Currently, lower terminals fail. ref: https://github.com/Kotlin/kotlinx-datetime#using-in-your-projects
+        # Currently, no API 30 virtual image
+        api-level: [26, 27, 28, 29]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
```

Comment:
As a result of trial and error, this seems to be efficient, but it may be better to adjust.